components/vf-design-tokens/gulpfile.js

```diff
@@ -106,7 +114,17 @@ gulp.task("tokens2:json", () =>
 );
 
 gulp.task("tokens:props", () =>
-  gulp.src(["./src/maps/*.yml"])
+  gulp.src([
+    "./src/maps/vf-colors.yml",
+    "./src/maps/vf-spacing.yml",
+    "./src/maps/vf-themes.yml",
+    "./src/maps/vf-ui-colors.yml",
+    "./src/maps/vf-zindex.yml",
+    "!./src/maps/vf-color__background.yml",
```

Comment:
Is this so they don't get generated as CSS custom props in the CSS?

Reply:
Yes, I think as they're 'design decisions' they're not something that would necessarily be benefited by being CSS custom properties.

I like the idea of -- if it's something that can be changed by the something in the browser (viewport size for example) or by a content creator (inside a CMS) then it should be a CSS custom property, but if it's essentially a 'static value' then Sass can pick up that Dx weight.